web/app/js/components/Paths.jsx

```diff
@@ -0,0 +1,91 @@
+import ConduitSpinner from "./ConduitSpinner.jsx";
+import ErrorBanner from './ErrorBanner.jsx';
+import { processRollupMetrics } from './util/MetricUtils.js';
+import React from 'react';
+import TabbedMetricsTable from './TabbedMetricsTable.jsx';
+import { ApiHelpers, urlsForResource } from './util/ApiHelpers.js';
+import 'whatwg-fetch';
+
+export default class Paths extends React.Component {
+  constructor(props) {
+    super(props);
+    this.api = ApiHelpers(this.props.pathPrefix);
+    this.handleApiError = this.handleApiError.bind(this);
+    this.loadFromServer = this.loadFromServer.bind(this);
+    this.state = this.initialState();
+  }
+
+  componentDidMount() {
+    this.loadFromServer();
+    this.timerId = window.setInterval(this.loadFromServer, this.state.pollingInterval);
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerId);
+  }
+
+  initialState() {
+    return {
+      lastUpdated: 0,
+      pollingInterval: 10000,
+      metricsWindow: "1m",
```

Comment:
I didn't put this at 10m because my api server was having a lot of trouble serving that request. will revert to 10m pending investigation of the api.